code/modules/clothing/suits/utility.dm

```diff
@@ -107,7 +107,8 @@
 	name = "radiation hood"
 	icon_state = "rad"
 	desc = "A hood with radiation protective properties. The label reads, 'Made with lead. Please do not consume insulation.'"
-	flags = BLOCKHAIR|THICKMATERIAL
+	flags = THICKMATERIAL
+	flags_inv = HIDEMASK|HIDEEARS|HIDEFACE|HIDEEYES|HIDEHAIR|HIDEFACIALHAIR

```

Comment:
Afaik it's only the rad and bio hoods that are slightly transparent. I think it's better to hide the mask and glasses to be honest, having gar glasses poking out of the hood is silly. We can always remove the transparency if it bothers people but as it is you barely notices the eyes through the hood.